crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__schema_sdl_export.snap

```diff
@@ -62,7 +62,7 @@ type ActiveJwkEdge {
 	cursor: String!
 }
 
-type Address implements ObjectOwner {
+type Address implements Iowner {
```

Comment:
nit IOwner

Reply:
The usual trick with `#[graphql(name = "IOwner")]` seemed to have made it work.